transport/transport-runtime/src/main/java/com/google/android/datatransport/runtime/scheduling/jobscheduling/WorkManagerSchedulerWorker.java

```diff
@@ -0,0 +1,58 @@
+// Copyright 2024 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package com.google.android.datatransport.runtime.scheduling.jobscheduling;
+
+import android.content.Context;
+import android.util.Base64;
+import androidx.annotation.NonNull;
+import androidx.work.Worker;
+import androidx.work.WorkerParameters;
+import com.google.android.datatransport.runtime.TransportContext;
+import com.google.android.datatransport.runtime.TransportRuntime;
+import com.google.android.datatransport.runtime.util.PriorityMapping;
+
+public class WorkManagerSchedulerWorker extends Worker {
+  private WorkerParameters params;
+
+  public WorkManagerSchedulerWorker(
+      @NonNull Context context, @NonNull WorkerParameters workerParams) {
+    super(context, workerParams);
+    this.params = workerParams;
```

Comment:
Why do you need to get the `params` around? based on the docs 

https://developer.android.com/develop/background-work/background-tasks/persistent/getting-started/define-work?_gl=1*kp2uzt*_up*MQ..*_ga*MTEyOTA3NjY2MS4xNzI1MDQzNjQz*_ga_6HH9YJMN9M*MTcyNTA0MzY0Mi4xLjAuMTcyNTA0MzY0Mi4wLjAuMA..

you should be able to get the input data directly by calling `getInputData()` directly from within the Worker